Assets/Game/Scripts/CharacterInfo.cs

```diff
@@ -13,15 +13,18 @@ public class CharacterInfo : MonoBehaviour {
 	// More information on character stats https://github.com/warpfox/EB-Clone/wiki/Character-Stats
 	[SerializeField] private int iq, guts, luck, vitality, speed, offense, defense;
 	[SerializeField] private int characterLevel, currentEXP, maxHP, maxPP, currentHP, currentPP;
-    private int offGrowthRate, defGrowthRate, speedGrowthRate, gutsGrowthRate, vitGrowthRate, iqGrowthRate, luckGrowthRate;
+  private int[] nessGrowthRates = new int[]{18,5,4,7,6,5,5};
+	private int[] paulaGrowthRates = new int[]{12,3,8,5,5,2,7};
+	private int[] jeffGrowthRates = new int[]{10,6,5,5,4,3,9};
+	private int[] pooGrowthRates = new int[]{21,18,7,3,3,4,4};
 
     // Use this for initialization
     void Start () {
 		// Testing adding items to inventory
 
 		Item item0 = new Item { ItemType = ItemType.Arms, ItemName = "Cheap Bracelet" };
-		Item item1 = new Item { ItemType = ItemType.Consumable, ItemName = "Bread Roll" };
-		Item item2 = new Item { ItemType = ItemType.Reusable, ItemName = "For Sale Sign" };
+		Item item1 = new Item { ItemType = ItemType.Food, ItemName = "Bread Roll", RecoversHP = 8 };
```

Comment:
I had removed the "RecoversHP" property from the Item class in one of my commits bc not all Items recover HP. I was  thinking we should keep a base class Item which has all common properties, and then there would be HealingItem : Item which can have a RecoversHP property.